command/connect/envoy/bootstrap_config.go

```diff
@@ -845,7 +845,7 @@ func appendHCPMetricsConfig(args *BootstrapTplArgs, hcpMetricsBindSocketDir stri
 		  "transport_api_version": "V3",
 		  "grpc_service": {
 			"envoy_grpc": {
-			  "cluster_name": "hcp_metrics_collector"
+			  "cluster_name": "metrics_collector"
```

Comment:
How come the naming is different here from `consul-telemetry-collector`?

What if this was something like "consul_telemetry_collector_loopback"?

Reply:
SGTM!

Reply:
yep that's better, will change it